lib/govuk_sidekiq/sidekiq_initializer.rb

```diff
@@ -12,7 +12,7 @@ def self.setup_sidekiq(govuk_app_name, redis_config = {})
       )
 
       Sidekiq.configure_server do |config|
-        config.log_formatter = Sidekiq::Logger::Formatters::JSON.new
+        config.log_formatter = Sidekiq::Logger::Formatters::JSON.new if ENV["GOVUK_SIDEKIQ_JSON_LOGGING"]
```

Comment:
May have forgotten the context on this - do we intend to keep this optional or was this while we were testing?

Reply:
Yup, we should keep this optional so that in dev you get nice logging rather than a JSON blob.